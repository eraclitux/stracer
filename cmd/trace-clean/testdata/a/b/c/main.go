package main

import (
	"fmt"

	"github.com/eraclitux/trace"
)

type dummy struct {
	Name string
}

func main() {
	trace.PrettyPrint("dummy:", dummy{"John"})
	trace.Printf("%s", "debug this!")
	fmt.Println("printed")
}
