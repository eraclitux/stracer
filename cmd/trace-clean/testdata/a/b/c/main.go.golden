package main

import (
	"fmt"

)

type dummy struct {
	Name string
}

func main() {
	fmt.Println("printed")
}
